src/main/java/org/sagebionetworks/web/client/place/ACTDataAccessSubmissionDashboardPlace.java

```diff
@@ -16,7 +16,7 @@ public String toToken() {
     return token;
   }
 
-  @Prefix("!ACTDataAccessSubmissionDashboard")
+  @Prefix("ACTDataAccessSubmissionDashboard")
```

Comment:
This dashboard uses a combination of GWT router + React Router, so we should use extra care to validate functionality in this place